### PR TITLE
 .build.yml: Build lomiri-url-dispatcher from source during CI builds.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

File: .build.yml [hunk 1]
```diff
@@ -187,6 +151,28 @@ before_scripts:
...
+  - cd lomiri-schemas-build
+  - cmake . -DCMAKE_INSTALL_PREFIX=/usr
+  - make
+  - make install
+  - cd -
+  - rm -Rf lomiri-schemas-build/
+  -
+  - cd ${START_DIR}
+  - if [ ! -d lomiri-url-dispatcher-build ]; then
+  -     git clone --depth 1  https://github.com/ubports/core/lomiri-url-dispatcher.git lomiri-url-dispatcher-build
```
Comment (tari01) [inline .build.yml:168]: https://gitlab.com/ubports/core/lomiri-url-dispatcher.git

**Review by tari01 — CHANGES_REQUESTED** [review]

File: .build.yml (hunk 1 above)
Comment (sunweaver) [reply .build.yml:168]: Amended.

Comment (tari01) [reply .build.yml:168]: It's github again :-)

**Review by tari01 — CHANGES_REQUESTED** [review]

**Comment by sunweaver** [comment]
@tari01: should now be ok... Sorry for the hassle.

**Review by tari01 — CHANGES_REQUESTED** [review]
Building `lomiri-url-dispatcher` requires `liblomiri-app-launch-dev`. It needs to be added as a temporary dependency.

UPDATE: I've just tried adding **lomiri-app-launch**, but that one again has dependecies, so we are in dependency-hell with this. Maybe we should disable this for Ubuntu and have it only for Debian, pulling from Debian repositories?

**Comment by sunweaver** [comment]
@tari01 I have decided to cancel this PR #75. The dependency pile behind lomiri-url-dispatcher is far to big (lomiri-app-launch, lomiri-api, mir, wlcs). Nothing we should do during CI builds if the distro doesn't provided that stack of dependencies.

**Pull request closed without merging**